controllers/controller.js

```diff
@@ -0,0 +1,34 @@
+//'use strict';
+
+
+// var mongoose = require('mongoose'),
+//   Product = mongoose.model('ProductSchema'),
+//   User= mongoose.model('UserSchema');
+
+  const Product = require('../models/ProductModel');
+  const User = require('../models/UserModel');
```

Comment:
separate each controller in separate file to be like this productsController.js and userController.js
and exporting your controllers should be like this 
```
const myControllerFunc = (res, req) => {
    // do what ever logic here
}

module.exports = {
    myControllerFunc,
    anotherControllerFunc,
}
```